setup.cfg

```diff
@@ -5,3 +5,7 @@ all_files  = 1
 
 [upload_sphinx]
 upload-dir = docs/_build/html
+
+[flake8]
+exclude = __init__.py
+max-line-length = 88
```

Comment:
MNE uses this:

https://github.com/mne-tools/mne-python/blob/master/setup.cfg#L19

Reply:
E241: [Multiple spaces after ','](https://github.com/PyCQA/pycodestyle/issues/4)
E305: [Expected 2 blank lines after end of function or class](https://lintlyci.github.io/Flake8Rules/rules/E305.html)
W504: [Line break occurred after a binary operator](https://lintlyci.github.io/Flake8Rules/rules/W504.html)

Reply:
I kinda like E305 so I would keep it but I agree with the rest. Thanks a lot for the suggestion @agramfort